zproject/urls.py

```diff
@@ -116,7 +116,7 @@
 
     # Login/registration
     url(r'^register/$', zerver.views.registration.accounts_home, name='register'),
-    url(r'^login/$',  zerver.views.auth.login_page, {'template_name': 'zerver/login.html'}, name='zerver.views.auth.login_page'),
+    url(r'^login/$',  custom_login, {'template_name': 'zerver/login.html'}, name='zerver.views.auth.login_page'),
```

Comment:
Please use the absolute path to avoid errors. It should have been `zerver.views.custom_login.custom_login`.